app/role_data.go

```diff
@@ -245,6 +245,12 @@ func addRoles(management *config.ManagementContext) (string, error) {
 	rb.addRoleTemplate("View Project Members", "projectroletemplatebindings-view", "project", true, false, false).
 		addRule().apiGroups("management.cattle.io").resources("projectroletemplatebindings").verbs("get", "list", "watch")
 
+	rb.addRoleTemplate("Manage Project Catalogs", "projectcatalogs-manage", "project", true, false, false).
+		addRule().apiGroups("management.cattle.io").resources("projectcatalogs").verbs("*")
+
+	rb.addRoleTemplate("View Project Catalogs", "projectcatalogs-view", "project", true, false, false).
+		addRule().apiGroups("management.cattle.io").resources("projectcatalogs").verbs("*")
+
```

Comment:
Does any project user get the role "Manage Project Catalogs" - or should we be adding this role only to a Project "owner"?

Reply:
I feel that a project member should be able to manage these, since we will allow creation of catalog at the project level. Plus the owner while adding members can deselect this from custom roles if some members shouldn't be able to manage the project catalogs

Reply:
@cjellick ^^ rbac decision needed